package.json

```
@@ -69,13 +70,14 @@
         "@ew-did-registry/proxyidentity": "0.5.2-alpha.90.0",
         "@gnosis.pm/safe-apps-sdk": "1.0.3",
         "@metamask/detect-provider": "^1.2.0",
-        "@walletconnect/web3-provider": "1.0.0-rc.4",
+        "@walletconnect/web3-provider": "^1.6.5",
         "axios": "^0.21.1",
         "base64url": "^3.0.1",
         "concurrently": "^6.1.0",
         "eth-ens-namehash": "^2.0.8",
         "ethers": "^5.4.4",
         "js-sha3": "^0.8.0",
+        "jsonwebtoken": "^8.5.1",
```

Instead of `jsonwebtoken` package, I think it would be better if we used the JWT package `ew-did-registry`
```suggestion
        "jsonwebtoken": "^8.5.1",
```